stable/sematext-docker-agent/templates/daemonset.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1beta2
```

Comment:
Can you set this up to conditionally install the daemonset when a key is provided? This is how we have others, such as datadog, doing it today. You can see an example [here](https://github.com/kubernetes/charts/blob/6814a5f8805a8c72140db1c7a470458a2049b7be/stable/datadog/templates/daemonset.yaml#L2).

Reply:
Sure, done. Our case is a little bit different because we have two token types. So, if any of them is supplied it will work, or both.